tests/conftest.py

```diff
@@ -29,8 +29,8 @@
 from invenio_vocabularies.records.api import Vocabulary
 
 
-@pytest.fixture(scope="module")
-def app_config(app_config):
+@pytest.fixture(scope="module", name="app_config")
+def fixture_app_config(app_config):
```

Comment:
Why the `fixture_` prefix? The decorator already declares it as a fixture :thinking:

Reply:
I added pylint. pylint is complaining that app_config within the function is overriding the outer one. see [here](https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/redefined-outer-name.html)